packages/api/tests/models/test-access-tokens-model.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+const test = require('ava');
+const { randomString } = require('@cumulus/common/test-utils');
+const {
+  fakeAccessTokenFactory,
+  fakeUserFactory
+} = require('../../lib/testUtils');
+const { RecordDoesNotExist } = require('../../lib/errors');
+const { AccessToken, User } = require('../../models');
+
+let accessTokenModel;
+let userModel;
+test.before(async () => {
+  process.env.AccessTokensTable = randomString();
+  process.env.UsersTable = randomString();
+
+  accessTokenModel = new AccessToken();
+  userModel = new User();
+
+  await accessTokenModel.createTable();
+  await userModel.createTable();
+});
+
+test.after.always(async () => {
+  await accessTokenModel.deleteTable();
+  await userModel.deleteTable();
+});
+
+test('AccessToken model sets the tableName from a param', (t) => {
+  const tableName = randomString();
+  const testAccessTokenModel = new AccessToken({ tableName });
+
+  t.is(testAccessTokenModel.tableName, tableName);
+});
+
+test('AccessToken model sets the table name from the AccessTokensTable environment variable', (t) => {
+  const envTableName = randomString();
+  process.env.AccessTokensTable = envTableName;
+
+  const testAccessTokenModel = new AccessToken();
+  t.is(testAccessTokenModel.tableName, envTableName);
+});
+
+test('create() creates a valid access token record', async (t) => {
+  const userRecord = fakeUserFactory();
+  await userModel.create(userRecord);
+
+  const accessTokenData = fakeAccessTokenFactory({ username: userRecord.userName });
+  const accessTokenRecord = await accessTokenModel.create(accessTokenData);
+
+  t.is(accessTokenRecord.accessToken, accessTokenData.accessToken);
+  t.is(accessTokenRecord.refreshToken, accessTokenData.refreshToken);
+  t.is(accessTokenRecord.username, userRecord.userName);
+  t.truthy(accessTokenRecord.expirationTime);
+});
```

Comment:
What about two tests, showing you can create valid records with or without a username?  Is that useful at all?

```
test('create() creates a valid access token record when provided a username', async (t) => {
  const username = 'username';
  const accessTokenData = fakeAccessTokenFactory({ username });
  const accessTokenRecord = await accessTokenModel.create(accessTokenData);

  t.is(accessTokenRecord.accessToken, accessTokenData.accessToken);
  t.is(accessTokenRecord.refreshToken, accessTokenData.refreshToken);
  t.is(accessTokenRecord.username, username);
  t.truthy(accessTokenRecord.expirationTime);
});

test('create() creates a valid access token record without a username', async (t) => {

  const accessTokenData = {
    accessToken: randomString(),
    refreshToken: randomString(),
    expirationTime: Date.now()
  };

  const accessTokenRecord = await accessTokenModel.create(accessTokenData);

  t.is(accessTokenRecord.accessToken, accessTokenData.accessToken);
  t.is(accessTokenRecord.refreshToken, accessTokenData.refreshToken);
  t.is(accessTokenRecord.username, undefined);
  t.truthy(accessTokenRecord.expirationTime);
});
```

Reply:
I think the existing test is already covering the case of creating one with a username/expiration. But since we were also creating access token records without those fields, it probably would be good to have a test for that, so I'll add it